xcodeproj/internal/xcode_schemes.bzl

```
@@ -6,6 +6,15 @@ load(":xcode_schemes_internal.bzl", "xcode_schemes_internal")
 
 _DEFAULT_BUILD_CONFIGURATION_NAME = "Debug"
 
+
+def _focus_actions(build_targets, actions):
+    focussed_actions = []
+    build_target_labels = [target.label for target in build_targets]
+    for action in actions:
+        if action.target in build_target_labels:
+            focussed_actions.append(action)
+    return focussed_actions
```

Super nitty, but the admittedly US-centric `focused` spelling is used elsewhere in this repo.

```suggestion
    focused_actions = []
    build_target_labels = [target.label for target in build_targets]
    for action in actions:
        if action.target in build_target_labels:
            focused_actions.append(action)
    return focused_actions
```